sensor/CountryIntelPlugin.js

```diff
@@ -0,0 +1,74 @@
+/*    Copyright 2021 Firewalla LLC
+ *
+ *    This program is free software: you can redistribute it and/or  modify
+ *    it under the terms of the GNU Affero General Public License, version 3,
+ *    as published by the Free Software Foundation.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+'use strict';
+
+const log = require('../net2/logger.js')(__filename);
+const Sensor = require('./Sensor.js').Sensor;
+const cc = require('../extension/cloudcache/cloudcache.js');
+const country = require('../extension/country/country.js');
+const zlib = require('zlib');
+const fs = require('fs');
+const Promise = require('bluebird');
+const inflateAsync = Promise.promisify(zlib.inflate);
+Promise.promisifyAll(fs);
+const Buffer = require('buffer').Buffer;
+const hashData = [{
+    hashKey: "mmdb:ipv4",
+    dataPath: `${__dirname}/../extension/country/data/geoip-country.dat`
```

Comment:
do not put any files under ~/firewalla, it's likely to be cleaned when running. put it under ~/.firewalla